jenkins/opensearch/benchmark-pull-request.jenkinsfile

```diff
@@ -179,6 +188,26 @@ pipeline {
                 printContributedVariables: true,
                 printPostContent: true
             )
+
+            parameterizedCron '''
+                H 4 * * * %DISTRIBUTION_URL=https://artifacts.opensearch.org/snapshots/core/opensearch/2.16.0-SNAPSHOT/opensearch-min-2.16.0-SNAPSHOT-linux-x64-latest.tar.gz;DISTRIBUTION_VERSION=2.16.0;TEST_WORKLOAD=big5;SINGLE_NODE_CLUSTER=true;DATA_INSTANCE_TYPE=r5.xlarge;USE_50_PERCENT_HEAP=true;USER_TAGS=run-type:nightly,cluster-config:x64-r5.xlarge-1-shard-0-replica-snapshot-baseline;WORKLOAD_PARAMS={"snapshot_repo_name":"benchmark-workloads-repo","snapshot_bucket_name":"benchmark-workload-snapshots","snapshot_region":"us-east-1","snapshot_base_path":"workload-snapshots","snapshot_name":"big5_1_shard"};TEST_PROCEDURE=restore-from-snapshot;CAPTURE_NODE_STAT=true;MIN_DISTRIBUTION=true
```

Comment:
Why do we need cron if all this is to trigger pull request runs?
Maybe rename the jenkinsfile if there is other use cases?

Thanks.

Reply:
This cron runs the benchmark against daily build distributions. 
This will allow the users to compare their pull request benchmark data against already merged commits.

Reply:
Got it.